.ci/common.sh

```diff
@@ -48,7 +50,16 @@ check_core_dumps() {
 }
 
 setup_deps() {
+	sudo pip install --upgrade pip
 	sudo pip install neovim
+
+	# For pip3
+	# https://github.com/travis-ci/travis-ci/issues/1528
+	# sudo apt-get install -q python3.3-dev
+	# curl -Ss http://python-distribute.org/distribute_setup.py | sudo python3
+	# curl -Ss https://raw.github.com/pypa/pip/master/contrib/get-pip.py | sudo python3
+	# sudo pip3.3 install neovim
```

Comment:
See https://github.com/Shougo/neovim/pull/4.

It appears to work by adding another entry to the matrix for "python: 3.4".